### PR TITLE
Update default xml for MailChimp

### PR DESCRIPTION
**Pull request by HaoZeke**

### CODE REVIEW

**Comment by HaoZeke** [comment]
Any updates on this?

**Comment by pup** [comment]
@HaoZeke If you could rebase `master` on top of this (e.g. updating this branch to the current state of master), I'd be more than happy to land this!

File: lib/jekyll-feed/feed.xml [hunk 1]
```diff
@@ -74,7 +74,7 @@
         {% unless post_image contains "://" %}
           {% assign post_image = post_image | absolute_url | xml_escape  %}
         {% endunless %}
-        <media:thumbnail xmlns:media="http://search.yahoo.com/mrss/" url="{{ post_image }}" />
+        <media:content xmlns:media="http://search.yahoo.com/mrss/" url="{{ post_image }}" medium="image" />
```
Comment (pathawks) [inline lib/jekyll-feed/feed.xml]: What is this about? Can you link to some documentation explaining this?

I don’t want to get rid of `media:thumbnail`, it’s used by feed readers.

**Comment by HaoZeke** [comment]
So the current layout isn't picked up by mailchimp which expects media:content
I could also leave thumbnail in there too, good point.. @pathawks

I'll update this sometime soon.

**Comment by pathawks** [comment]
You could maybe put them both in the same `media:group`?

We may have to do some experiments to see how this will be rendered in various feed readers.

**Comment by DirtyF** [comment]
Fixed in #290

**Pull request closed without merging**